src/styles/blocks/nav.scss

```
@@ -0,0 +1,47 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    list-style: none;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    text-decoration: none;
+    height: 60px;
```

Use padding instead of hardcodding height
```suggestion
    padding: 23px 0;
```

---

done